inspirehep/dojson/hep/fields/bd5xx.py

```diff
@@ -209,8 +209,12 @@ def funding_info2marc(self, key, value):
 @utils.filter_values
 def license(self, key, value):
     """Add Terms Governing Use and Reproduction Note."""
+    license_value = force_force_list(value.get('a'))
+    # We strip away redundant 'Open Access' string
+    license_value = [val for val in license_value if license_value != 'Open Access']
+    license_value = force_single_element(license_value)
```

Comment:
I have checked. Only 17 records have these Open Access license (which is not a valid name for a license) and have also a valid CC-BY-* one. These are also the very same records having more than one license.

All the other papers have valid CC-* licenses.